app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -362,7 +362,9 @@ open class DefaultToolbarMenu(
             BrowserMenuDivider(),
             reportSiteIssuePlaceholder,
             findInPage,
+            BrowserMenuDivider(),
             getSetDefaultBrowserItem(),
+            BrowserMenuDivider(),
```

Comment:
If the item is not shown the divider will show either way.

Reply:
As `getSetDefaultBrowserItem()` returns null when the experiment is not active

Reply:
Good catch. I am showing them depending on the value now. Not sure, but I think one of them should be shown either way.

Reply:
👍🏽